tests/test_create.py

```diff
@@ -2177,94 +2177,105 @@ def side_effect(self, url, **kwargs):
 
 
 @pytest.mark.skipif(on_win, reason="python doesn't have dependencies on windows")
-def test_disallowed_packages():
-    with make_temp_env() as prefix:
-        with env_var(
-            "CONDA_DISALLOWED_PACKAGES",
-            "sqlite&flask",
-            stack_callback=conda_tests_ctxt_mgmt_def_pol,
-        ):
-            with pytest.raises(CondaMultiError) as exc:
-                run_command(Commands.INSTALL, prefix, "python")
+def test_disallowed_packages(
+    tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture, monkeypatch: MonkeyPatch
+):
+    with tmp_env() as prefix:
+        monkeypatch.setenv("CONDA_DISALLOWED_PACKAGES", "sqlite&flask")
+        reset_context()
+        with pytest.raises(CondaMultiError) as exc:
+            conda_cli("install", f"--prefix={prefix}", "python", "--yes")
         exc_val = exc.value.errors[0]
         assert isinstance(exc_val, DisallowedPackageError)
         assert exc_val.dump_map()["package_ref"]["name"] == "sqlite"
 
 
-def test_dont_remove_conda_1():
-    pkgs_dirs = context.pkgs_dirs
-    prefix = make_temp_prefix()
-    with env_vars(
-        {"CONDA_ROOT_PREFIX": prefix, "CONDA_PKGS_DIRS": ",".join(pkgs_dirs)},
-        stack_callback=conda_tests_ctxt_mgmt_def_pol,
-    ):
-        with make_temp_env(prefix=prefix):
-            _, _, _ = run_command(Commands.INSTALL, prefix, "conda", "conda-build")
-            assert package_is_installed(prefix, "conda")
-            assert package_is_installed(prefix, "pycosat")
-            assert package_is_installed(prefix, "conda-build")
+def test_dont_remove_conda_1(
+    monkeypatch: MonkeyPatch, tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture
+):
+    with tmp_env() as prefix:
+        monkeypatch.setenv("CONDA_ROOT_PREFIX", prefix)
+        reset_context()
+        assert context.root_prefix == str(prefix)
+        _, _, _ = conda_cli(
```

Comment:
I like to leave a little extra space and no need to store the output if we don't use it:

```suggestion

        conda_cli(
```